tool/mapeditor.re

```diff
@@ -155,6 +156,8 @@ fn create_map_editor_input_ui() -> Ui {
                     ElementType::Lift
                 } else if input.contains("pi") {
                     ElementType::Pipe
+                } else if input.contains("spr") {
```

Comment:
What about just `s`? Also you need to add `s` and `pi` to the error text and the menu text.